src/data/polynomial/laurent.lean

```diff
@@ -314,6 +314,13 @@ left_inverse_trunc_to_laurent.injective
   f.to_laurent = g.to_laurent ↔ f = g :=
 ⟨λ h, polynomial.to_laurent_injective h, congr_arg _⟩
 
+lemma _root_.polynomial.to_laurent_ne_zero (f : R[X])  (f0 : f ≠ 0) :
+  f.to_laurent ≠ 0 :=
```

Comment:
This is true as an iff, right?

Reply:
Yes, I changed the statement and proved the `iff` version, thanks!